Gdbus unit tests do not pass on 0.48 rc

```
======================================================================
FAIL: test_introspect_dependencies (__main__.LinuxlikeTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./run_unittests.py", line 3797, in test_introspect_dependencies
    self.assertEqual(t['filename'], 'gdbus/generated-gdbus-doc-' + ifile)
AssertionError: 'gdbus/generated-gdbus-doc-com.example.Sample.xml' != 'gdbus/generated-gdbus-doc-gdbus/data/com.example.Sample.xml'
- gdbus/generated-gdbus-doc-com.example.Sample.xml
+ gdbus/generated-gdbus-doc-gdbus/data/com.example.Sample.xml
?                           +++++++++++
```

This is a release blocker.

--- a/run_unittests.py
+++ b/run_unittests.py
@@ -3794,7 +3794,7 @@ class LinuxlikeTests(BasePlatformTests):
                 break
         self.assertIsInstance(docbook_target, dict)
         ifile = self.introspect(['--target-files', 'generated-gdbus-docbook@cus'])[0]
-        self.assertEqual(t['filename'], 'gdbus/generated-gdbus-doc-' + ifile)
+        self.assertEqual(t['filename'], 'gdbus/generated-gdbus-doc-' + os.path.basename(ifile))
 
     def test_build_rpath(self):
         if is_cygwin():

